app/models.py

```diff
@@ -674,7 +674,7 @@ class ServiceSafelist(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), index=True, nullable=False)
-    service = db.relationship('Service', backref='whitelist')
+    service = db.relationship('Service', backref='safelist')
```

Comment:
What does this do? Does this need a DB upgrade?

Reply:
no, we're still using the `service_whitelist` database table here, but saying that it's joined to a service via `service.safelist` (ie `service.safelist` now gives the list of corresponding rows from the `service_whitelist` table.

Possibly I'm splitting things up a bit too much, but figured I'd make the db migration PR as small as possible since I haven't done one of them before 😬

Reply:
That makes sense. Thank you for the clarification!